### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by staukini** [alangecker/bigbluebutton-docker#80]
I'm currently trying to get the bbb service running on an iPhone.

The screen-share and the slides provided by bbb are working fine but when joining the audio (listen only or microphone) I'll get "Connection error ICE 1007" displayed on the screen,

I've tried to resolve this problem by myself and found that the problem probably relies on STURN, but in my current configuration (behand a NAT) the port 465 is used by the coturn docker instanz as the firewall logs shows. 

Has anyone got iPhones (probably also iPads) to work with bbb?

cheers

**Comment by staukini** [comment]
Update: 
I tried using an public stun server which I hoped would solve the problem. 
But even with a new "installation" of bbb and an public stun server I get the same ICE 1007 Error.

In the live logs of the firewall are no packages dropped.
I have setup NAT for TCP 80,443 3478(STUN), 465(SMTP) and UDP (for testing also TCP/UDP) 16384-32768.

I don't know what else should be the misconfiguration.

PC-Browser's are working fine - iOS makes troubbles (Android work's fine)

**Comment by cjhille** [comment]
> Has anyone got iPhones (probably also iPads) to work with bbb?

iOS Sessions (with iPhone and iPad) work fine for me. I'm using them regularly. I image it might be due to your NAT scenario. Have you tried on a host without NAT? [2.2.29](https://github.com/bigbluebutton/bigbluebutton/releases/tag/v2.2.29) introduced some config flag for "Ice" related errors. Have you tried this or other config options?

**Comment by staukini** [comment]
I tried now for a few days some proxy flags as you tried to guide me with the link.
But there i nether found a documented solution for the error code 1007 nor any other flag solved the problem....
Ohh boy - I'm running out of ideas...

**Comment by staukini** [comment]
As posted in #89 a complete reinstallation doesn't resolve this issue.

I assume #89 and this one have the same root cause.

**Comment by rasos** [comment]
Yes several people have problems with iOS. Fine tuning TURN Servers or not offering TURNS are some proposals https://github.com/bigbluebutton/bigbluebutton/issues/10746#issuecomment-728249466
